app/helpers/host_initiator_helper/textual_summary.rb

```diff
@@ -0,0 +1,48 @@
+module HostInitiatorHelper::TextualSummary
+  include TextualMixins::TextualGroupTags
+
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    TextualGroup.new(
+      _("Properties"),
+      %i[
+        name
+        ems
+        physical_storage
+      ]
+    )
+  end
+
+  def textual_group_san_addresses
+    san_addresses_values = []
+
+    @record.san_addresses.each do |san_address|
+      san_addresses_values << san_address.get_address_info
+    end
```

Comment:
The ruby way is to do it with a map instead of a loop, like this:
```
san_address_values = @record.san_addresses.map { |san_address| san_address.get_address_info }
```
Or if it's too complex for a one-liner, then like this:
```
san_address_values = @record.san_addresses.map do |san_address| 
   san_address.get_address_info
end
```